contract-v1/src/main.rs

```diff
@@ -49,20 +57,21 @@ pub extern "C" fn counter_get() {
         .unwrap_or_revert_with(ApiError::Read)
         .unwrap_or_revert_with(ApiError::ValueNotFound);
     let typed_result = CLValue::from_t(result).unwrap_or_revert();
-    runtime::ret(typed_result); // return the count value
+    runtime::ret(typed_result); // Return the count value.
 }
 
+// Entry point that executes automatically when a caller interacts with the contract. 
```

Comment:
We have talked with Ed & Karan about using phrases such as "install contract" and "interact with the contract", instead of "deploying a contract". So, I have updated the comment to say "when a caller installs the contract".